### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by 39hn**
intentionally left out some tech keywords and texts that would become too long or unnatural when translated.

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @39hn! I'm going to give the @elixirschool/korean translation team a chance to review before we merge 🎉

File: priv/gettext/ko/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -343,32 +343,32 @@ msgstr ""
...
 #: lib/school_house_web/live/conferences_live.html.heex:8
 msgid "Check out all of the conferences that focus on Elixir!"
-msgstr ""
+msgstr "Elixir 중심의 컨퍼런스들을 확인해보세요!"
 
 #, elixir-format
 #: lib/school_house_web/templates/page/podcasts.html.heex:8
 msgid "Check out all the great of Podcasts that focus on Elixir and the BEAM ecosystem!"
-msgstr ""
+msgstr "Elixir 와 BEAM 생태계 중심의 팟캐스트들을 확인해보세요!"
```
Comment (gidskql6671) [inline priv/gettext/ko/LC_MESSAGES/default.po:371]:
```suggestion
msgstr "Elixir와 BEAM 생태계 중심의 팟캐스트들을 확인해보세요!"
```
다른 번역체와의 통일성을 위해 띄어쓰기를 제거했습니다.

**Review by gidskql6671 — APPROVED** [review]
감사합니다 :)

**Comment by doomspork** [comment]
Thank you @39hn and @gidskql6671 🙏

**Pull request merged**